README.md

```diff
@@ -49,11 +49,22 @@ yarn dev
 
 Visit <http://localhost:3040/>
 
+To run and develop the module NodeJS 16.13.0 is required. 
+You can use nvm (Node version manager) in order to use the NodeJS version 16.13.0
+- [nvm](https://github.com/nvm-sh/nvm)
+- [nvm for windows](https://github.com/coreybutler/nvm-windows)
+
 ## Starting the backend
 
-In order to use the frontend together with the API for it you need to clone and start the backend project.
+In order to use the frontend together with the API for it you need to clone and start the backend project or you can use the API in production https://dev.podkrepi.bg/api/v1.
+
+To start the backend project follow API instructions at <https://github.com/podkrepi-bg/api#setup-development-environment>
 
-Follow API instructions at <https://github.com/podkrepi-bg/api#setup-development-environment>
+To use the API in production change .env.local.example
+```
+API_URL=http://podkrepi.bg/api/v1
+IMAGE_HOST=podkrepi.bg
```

Comment:
I think that this should always be localhost if you are running the frontend locally.